ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/workloadCveOverviewPage.test.js

```diff
@@ -76,4 +79,174 @@ describe('Workload CVE overview page tests', () => {
             */
         });
     });
+
+    describe('Images without CVEs view tests', () => {
+        const cveListRouteMatcherMap = {
+            getImageCVEList: {
+                method: 'POST',
+                url: `/api/graphql?opname=getImageCVEList`,
```

Comment:
i think we have an API endpoints constants file for cypress tests you can probably use? unless the endpoints are intentionally put here

Reply:
Ah I think you're right, there is at least a helper function for creating these gql endpoints. Let me see if I can find it and replace this.

Reply:
Bravo for intercepts. And for thought to use generic helper.

On other side of coin, I have moved as much container-specific routing out into container specific helpers.

Helper functions in WorkloadCves.helpers.js file might preclude concern about implementation details in test file.